### ISSUE TITLE
Ways to reach more of the community

### ISSUE THREAD

**Issue by lsr-explore** [w3c-cg/aar#7]
@brianelton asked during the last community group meeting how we could reach out to the community.   Thought I'd start an issue to get the discussion started...

Proposal
1) web-a11y.slack.com is a fairly active community slack group for the a11y community.   We could create a couple of channels here for discussions.
2) Create a linkedin group

What are some other ideas?

**Comment by brianelton** [comment]
Closing issue. The AAR CG will be starting a new direction soon.
